### PR TITLE
Add instructions how to modify Tm calculation parameters

### PR DESCRIPTION
**Pull request by spalemartin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -218,6 +217,11 @@ make run-monitor
 - [webserver](webserver): Main entrypoint for the user. Simple nginx proxy of requests to the application API and frontend.
 - [lambda](lambda): Provides AWS Lambda endpoint running Primer3 binary locally in a Docker container by [SAM CLI](https://github.com/awslabs/aws-sam-cli).
 
+More information can be found in the respective **README** files:
+- backend: [README.md](backend/README.md)
```
Comment (pmejzlik) [inline README.md:221]: Here we could mention that the backend README covers changing parameters for Tm calculation.

File: backend/README.md [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+```
+
+For PAS oligo Tm’s a Tm correction factor of 3 °C was applied.
+
+The buffer conditions are set to support Q5 reactions. However, the minimum and maximum reaction Tm is fully adjustable, so users that know their Tm range can set the optimal parameters to reflect their reaction conditions.
+
+
+### Changing Tm calculation parameters
+
+There's a class `TemperatureConfig` that can be instantiated with different values of the abovementioned parameters.
```
Comment (pmejzlik) [inline backend/README.md:37]: temperature_calculator.py contains class ...

File: backend/README.md [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+    tris = FloatProperty(default=20)
+    mg = FloatProperty(default=2)
+    dntp = FloatProperty(default=0.2)
+    precision = IntegerProperty(default=0)
+)
+```
+
+### Allowed paramater values:
+
+Th following section describes the range of available values for some of the parameters above.
```
Comment (pmejzlik) [inline backend/README.md:58]: The following section ...

File: README.md (hunk 1 above)
Comment (spalemartin) [reply README.md]: Mention added.

File: backend/README.md (hunk 2 above)
Comment (spalemartin) [reply backend/README.md]: Resolved.

File: backend/README.md (hunk 3 above)
Comment (spalemartin) [reply backend/README.md]: Fixed.

**Pull request merged**